### DISCUSSION TITLE
Are you still developing this program? If you're in trouble, ask us for help

### DISCUSSION THREAD

**Discussion by BUGProtectionAssociation** [DrewNaylor/guinget#157] [General]
You can get help from the developers of [wingetUI](https://github.com/martinet101/WingetUI/)
[He](https://github.com/martinet101) may could help you (some functions and interface translations of the program)

**Comment by DrewNaylor** [comment]
I'm still working on it, just taking a break for now because I'm working on YouTube videos.
